digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/ConversionDatabase.scala

```diff
@@ -30,11 +31,10 @@ object ConversionDatabase {
         listOfPoint.toList
       }
     }
-
   }
 
   lazy val dataSource = {
-    val cfg = new BoneCPConfig(PostGISDatabase.loadProperties("/conversion.bonecp.properties"))
+    val cfg = new BoneCPConfig(Digiroad2Properties.bonecpProperties)
```

Comment:
en nyt tiedä käytetäänkö tätä conversion hommaa enään mihinkään mut tän pitäis varmaan käytää silti eri yhteyttä, joku conversio yhteys

Reply:
Jos tämä on vielä käytössä, niin korjataan myöhemmin käyttämään oikeita tunnuksia (nyt annettu digiroad-tunnukset).